File: web-services/services/auditing/src/main/java/com/cryptocaddy/services/auditing/model/response/ResponseUserData.java
package com.cryptocaddy.services.auditing.model.response;

import java.util.ArrayList;
import java.util.List;

public class ResponseUserData {

    private List<ResponseExchangeWrapper> allExchangeWrappers;

    public ResponseUserData() {
        allExchangeWrappers = new ArrayList<>();
    }

    public List<ResponseExchangeWrapper> getAllExchangeWrappers() {
        return allExchangeWrappers;
    }

    public void setAllExchangeWrappers(List<ResponseExchangeWrapper> allExchangeWrappers) {
        this.allExchangeWrappers = allExchangeWrappers;
    }

    public void addExchangeWrapper(ResponseExchangeWrapper responseExchangeWrapper){
        allExchangeWrappers.add(responseExchangeWrapper);
    }
}

File: web-services/auditing/src/main/java/com/cryptocaddy/services/auditing/api/dto/response/ResponseUserData.kt
package com.cryptocaddy.services.auditing.api.dto.response

import java.util.ArrayList

class ResponseUserData {

    private var allExchangeWrappers: MutableList<ResponseExchangeWrapper>? = null

    init {
        allExchangeWrappers = ArrayList()
    }

    fun getAllExchangeWrappers(): List<ResponseExchangeWrapper>? {
        return allExchangeWrappers
    }

    fun setAllExchangeWrappers(allExchangeWrappers: MutableList<ResponseExchangeWrapper>) {
        this.allExchangeWrappers = allExchangeWrappers
    }

    fun addExchangeWrapper(responseExchangeWrapper: ResponseExchangeWrapper) {
        allExchangeWrappers!!.add(responseExchangeWrapper)
    }
}
